pages/apis/graphql/graphql_cookbook.md

```diff
@@ -596,6 +596,114 @@ mutation {
 }
 ```
 
+## Clusters
+
+A collection of common tasks with clusters using the GraphQL API.
+
+### List cluster IDs
+
+Get the first 10 clusters and their information for a particular organization:
+
+```graphql
+query getClusters {
+  organization(slug: "organization-slug") {
+    clusters(first: 10){
+      edges{
+        node{
+          id
+          uuid
+          color
+          description
+        }
+      }
+    }
+  }
+}
+```
+
+### List cluster queue IDs
+
+Get the first 10 cluster queues for a particular cluster by specifying its UUID in `cluster-uuid`:
+
+```graphql
+query getClusterQueues {
+  organization(slug: "organization-slug") {
+    cluster(id: "cluster-uuid") {
+      queues(first: 10) {
+        edges {
+          node {
+            id
+            uuid
+            key
+            description
+          }
+        }
+      }
+    }
+  }
+}
+```
+
+### List jobs in a particular cluster queue
+
+To get jobs for a particular cluster queue, use the `clusterQueue` filter, passing in the ID of the cluster queue to filter jobs from:
+
+```graphql
+query getClusterQueueJobs {
+  organization(slug: "organization-slug") {
+    jobs(first: 10, clusterQueue: "cluster-queue-id") {
+      edges {
+        node {
+          ... on JobTypeCommand {
+            id
+            state
+            label
+            url
+            build {
+              number
+            }
+            pipeline {
+              name
+            }
+          }
+        }
+      }
+    }
+  }
+}
+```
+
+To obtain jobs for a particular cluster queue in a particular state, use the `clusterQueue` filter, passing in the ID of the cluster queue to filter jobs from, and the `state` list filter by one or more [JobStates](https://buildkite.com/docs/apis/graphql/schemas/enum/jobstates):
```

Comment:
Style: Again just removing a usage of "particular" to keep it where I think it's more needed.

```suggestion
To obtain jobs for a cluster queue in a particular state, use the `clusterQueue` filter, passing in the ID of the cluster queue to filter jobs from, and the `state` list filter by one or more [JobStates](https://buildkite.com/docs/apis/graphql/schemas/enum/jobstates):
```

Reply:
TIA @mbelton-buildkite - amended 👍